h2o-py/h2o/h2o.py

```diff
@@ -591,6 +593,7 @@ def parse_setup(raw_frames, destination_frame=None, header=0, separator=None, co
     :param na_strings: A list of strings, or a list of lists of strings (one list per column), or a dictionary
         of column names to strings which are to be interpreted as missing values.
     :param skipped_columns: an integer lists of column indices to skip and not parsed into the final frame from the import file.
+    :param non_data_line_markers: A string containing custom characters to be treated as non-data line markers
```

Comment:
this is confusing, isn't it? 

I would expect that `non_data_line_markers = None` means there are no such markers, not that the default will be applied

Reply:
it also may be confusing what a non-data-line-marker is, I would suggest rewording, to something like:

If a line in imported file starts with any character in given string it will be imported. Empty string means all lines are imported, None means that default behaviour for given format will be used.

Reply:
Yes, it is confusing, but as someone hard-coded these values earlier, it is harder to change it right now. So, should we introduce a new default being handed over ? Some special combination of chars, like `DEFAULT` or something like that ? Seems also confusing to the users.

Best solution would be to drop the default behavior.

Reply:
One solution to keep the old behavior and to make it less cofusing is to named the parameter `custom_non_data_line_markers`.

Reply:
See commit: https://github.com/h2oai/h2o-3/pull/3380/commits/827331e88c90f0317ec303b23a80aaf5fa9a6470